MSAL/src/instance/MSALAccountsProvider.m

```diff
@@ -350,7 +350,7 @@ - (void)allAccountsFromDevice:(MSALAccountEnumerationParameters *)parameters
 {
     BOOL shouldCallBroker = NO;
     
-    if (@available(iOS 13.0, macOS 10.15, *))
+    if (@available(macOS 10.15, *))
```

Comment:
Shouldn't this be `if (@available(iOS 14.0, macOS 10.15, *))` else the code inside the condition would not compile for iOS anymore? The code would only run on macOS since iOS was omitted

Reply:
i believe the  "*" indicates that the code is also for all other platforms beyond the ones listed . So it should be available for iOS.

Reply:
@Veena11 of course! Thanks :) Although is there API required that's no longer going to work on iOS 13? Xcode 14 seems to be otherwise okay with iOS 13 as a minimum target